spec/services/state_file/after_transition_messaging_service_spec.rb

```diff
@@ -101,6 +101,19 @@
     end
   end
 
+  describe "#send_efile_submission_terminal_rejected_message" do
+    let(:message) { StateFile::AutomatedMessage::TerminalRejected }
+
+    it "sends the accepted refund" do
+      expect do
+        messaging_service.send_efile_submission_terminal_rejected_message
+      end.to change(StateFileNotificationEmail, :count).by(1)
```

Comment:
should we have a `StateFileNotificationEmail` equivalent for texts?

Reply:
(i think the answer is probably yes)

Reply:
If you mean for the test case, sure I can add. If you mean the `AutomatedMessage` hierarchy may need to be refactored I'm not so sure. I'd need to think about it. I'll make an adjustment based on the former assumption though and let me know if you want to talk about moving around automated messages.

Reply:
copying over my slack message because i realized it's good to have the record in github:

"so sorry my comment about this wasn't clearer but what i meant about a StateFileNotificationEmail equivalent was: the record we keep for an outgoing email in FYST is StateFileNotificationEmail and since we didn't send texts last year we didn't make one for texts. but i think it would make sense to make a minimal model called StateFileNotificationTextMessage (based on the email one) and create a new one when we send text notifications. and if that feels like it's adding too much scope for this story i think a follow-up would be fine"